### PR TITLE
Updating tour guide speech rate to better use maptext

### PR DESCRIPTION
**Pull request by Azrun**
[bug - minor]
## About the PR 
Allows tourguide "presentations" to be further broken up to to be displayed as maptext.

Current Behavior:
atlas_tour_pre

PR Behavior:
atlas_tour_post

*  Original presentation in chat is maintained. 
image

* Add chat style for bots (Consolas looks great in chat but terrible as maptext)

Tourguide transition out of "presenting" is based on the tour text and process rate of guide.

Adjust all "neat_things" to be in a `spawn_dbg` to allow for desired timing and to help ensure `machines` `doWork` does not `sleep`.

## Why's this needed? 
Improve tour guide experience.

## Changelog

```changelog
(u)Azrun
(+)Improve tour guide floating chat display.
```

### CODE REVIEW

File: code/modules/robotics/bot/guardbot.dm [hunk 1]
```diff
@@ -3677,12 +3673,80 @@
...
+										break
+
+								speak_with_maptext(copytext(html_encode(tour_line), 1, MAX_MESSAGE_LEN), TRUE)
+								sleep(5 SECONDS)
+					else
+						speak_with_maptext(copytext(html_encode(current_tour_text), 1, MAX_MESSAGE_LEN), TRUE)
+
+
+#define MAPTEXT_SLICE_SIZE 100 // Reduce maptext slice size to avoid 4 rows
+
```
Comment (ZeWaka) [inline code/modules/robotics/bot/guardbot.dm:3718]:
```suggestion
```

File: code/modules/robotics/bot/guardbot.dm [hunk 2]
```diff
@@ -3677,12 +3673,80 @@
 
 			return
 
+		// take a text string and parse out pauses
+		proc/speak_with_pause(text, yield_to_neat=FALSE)
+			var/delays = 0
+			var/proc_delay = master.base_tick_spacing*(2**(master.processing_tier-1))
+
+			SPAWN_DBG(0) // do not delay doWork
```
Comment (ZeWaka) [inline code/modules/robotics/bot/guardbot.dm:3681]: :)

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/modules/robotics/bot/guardbot.dm (hunk 2 above)
Comment (Azrun) [reply code/modules/robotics/bot/guardbot.dm:3681]: waht

Comment (Azrun) [reply code/modules/robotics/bot/guardbot.dm:3681]: 🙀

**Review by ZeWaka — APPROVED** [review]

**Pull request merged**